lavuelib/imageSource.py

```diff
@@ -361,43 +359,31 @@ def getData(self):
         try:
             image = None
             metadata = ""
+            frame = 0
             try:
                 if self.__handler is None:
                     self.__handler = imageFileHandler.NexusFieldHandler(
                         str(self.__nxsfile))
                 if self.__node is None:
                     self.__node = self.__handler.getNode(self.__nxsfield)
-                    metadata = self.__handler.getMetaData(self.__node)
+                    try:
+                        metadata = self.__handler.getMetaData(self.__node)
+                    except Exception:
+                        metadata = ""
                     # if metadata:
                     #     print("IMAGE Metadata = %s" % str(metadata))
                 fid = self.__handler.getLastFrame(self.__node, self.__gdim)
-                if self.__nxslast:
-                    if fid - 1 > self.__frame or fid - 1 < self.__frame:
```

Comment:
Why -1? I think getLastFrame returns just the index of the last frame, not the count of the dataset.

Reply:
Hello @TerryGeng, 
Thank you for your PR. I have a few comment on it.
- "open from the last frame" have to state as it is. It was requested by our users. With this feature you can "play the  movie" of all your images from the first one to the last one. I would expect that the fame variable you add would represent the last frame to show  in the movie when the feature is on. If the feature is off  and start button in on it shows
the frame selected in the new widget. As you have noticed minus indices denote frames from the end so frame=-1 would correspond the the behavior without your PR.
- it would be good to have possibility to change frames after the "Start" button is pressed.
- with `getLastFrame` in principle you are right but the current indices inform us also when there is no frame in the image (returns 0). Therefore, it is better to change the method name to something like getNumberOfFrames or getFrameCounts.